sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/models/CertificatePolicy.java

```diff
@@ -62,7 +62,7 @@
      * published to certificate transparency logs.
      */
     @JsonProperty(value = "cert_transparency")
-    private Boolean certificateTransparency;
+    private Boolean certificateTransparent;
```

Comment:
Is this only changed because you changed the public API to `isCertificateTransparent` (different from other languages, but makes sense)?

Reply:
Actually, after reviewing notes I don't think this was supposed to change, or was that discussed offline?

Reply:
was discussed during the meeting that isCertificateTransparency getter sounds odd/weird.
Changed this idiomatically to read isCertificateTransparent for the getter.
was discussed and decided with java architect (@JonathanGiles).